Packs/ThreatIntel/Layouts/layoutscontainer-ThreatReportLayout.json

```diff
@@ -0,0 +1,717 @@
+{
+    "definitionId": "ThreatIntelReport",
+    "group": "incident",
```

Comment:
i dont think this should be the group

Reply:
`group` is a required field - without it, it throws the SDK out of order and the file recognition is incorrect.

Reply:
If definitionId exists and is not empty it takes priority over group - right @asafshen ?

Reply:
@roysagi you are correct, but I think its better to leave the group empty if possible, or at least give it a different value which is `incident`/`indicator` (maybe `generic`) 

this will reduce ambiguity for future readers

Reply:
@roysagi what about the comment above? this api clarity is important imo